src/libs/Network.js

```diff
@@ -9,7 +9,9 @@ import createCallback from './createCallback';
 import * as NetworkRequestQueue from './actions/NetworkRequestQueue';
 
 let isReady = false;
+let isOffline = false;
 let isQueuePaused = false;
+let persistedRequestsQueueRunning = false;
```

Comment:
The name comes from this usagage and readability reasons 
This method is **starting** something rather than making it **not paused**

```js
function flushPersistedRequestsQueue() {
    if (persistedRequestsQueueRunning) {
        return;
    }

    persistedRequestsQueueRunning = true;
```

vs

```js
function flushPersistedRequestsQueue() {
    if (!persistedRequestsQueuePaused) {
        return;
    }

   persistedRequestsQueuePaused = false;
```

**running** is the "special case" for this queue, while **paused** is the special case for the regular queue 

I'm reluctant to refactor `isQueuePaused` to `isMainQueueRunning` it would change usages and might affect their intent as well, also it kind a sound like we're online too